dts/bindings/mdio/xilinx,axi-ethernet-7.-2-mdio.yaml

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) 2024 CISPA Helmholtz Center for Information Security
+# SPDX-License-Identifier: Apache-2.0
+
+# Common fields for MDIO controllers
+
+include: mdio-controller.yaml
+
+compatible: xlnx,axi-ethernet-7.2-mdio
+
+description: MDIO interface of Xilinx AXI 1G/2.5G Ethernet Subsystem.
+
+properties:
+  reg:
+    required: true
+    type: array
+    description: |
```

Comment:
reg is a standard property should not be overriden in a binding

Reply:
Removed

Reply:
it's not removed